### DISCUSSION TITLE
discord,twitter alternative

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [oxen-io/session-desktop#2581] [Ideas]
session Pull Fork，Providing people with safe community networks
twitter --alternative-- https://github.com/nostr-protocol/nostr
discord  --alternative-- https://rvlt.gg

Onion routing through session layer 3

**Comment by JEDUMAR** [comment]
La plataforma Twitter tiene una interfaz mas amigable fácil de usar que Discordia eso es lo que ocurre puede ser sin duda una mejor opción pero la facilidad de uso no la hace apetecible... Yo estoy de acuerdo con la descentralización pero lo hacen muy complicado y es importante la simplicidad al momento de usar, así lo hacen las populares y es por ello que no las dejan de usar... Teniendo desventajas as siguen usando Whatsapp, Youtube, Twitter, y otras son mas simples de usar. hay esta el detalle.!
